projects/noloan/app/src/main/java/hasadna/noloan/mainactivity/MainActivity.java

```diff
@@ -162,17 +164,24 @@ public boolean onNavigationItemSelected(MenuItem item) {
   // Update the number of messages in the title - Called from recyclerViewer adapters when list
   // changes
   public void updateTitles() {
-    // Main title
-    statusTitle.setText(String.valueOf(SmsMessages.get().getDbMessages().size()));
+    // Main title - Suspicious messages found in the inbox similar to the spams in the DB (body &
+    // sender)
+    // Not necessarily the number of messages the user had suggested.
+    statusTitle.setText(String.valueOf(SmsMessages.get().countInboxSpam()));
 
-    // Tab's titles
+    // Number of messages in the inbox
     tabLayout
         .getTabAt(0)
         .setText(
             getString(R.string.inboxFragment_title, SmsMessages.get().getInboxMessages().size()));
+
+    // Number of messages that the user had suggested
```

Comment:
To be fixed next push. This isn't correct. Suppose to be number of messages relevant to the user's inbox.